docker-compose.yml

```diff
@@ -4,10 +4,10 @@ services:
   web:
     image: martel/mastermind-ui
     environment:
-      - MASTERMIND_API_HOST='localhost'
-      - MASTERMIND_API_PORT='3000'
-      - MASTERMIND_OAUTH_URI='account.lab.fiware.org'
-      - MASTERMIND_OAUTH_CLIENT_ID='f856da058c20414db0e946d234a5b9b1'
+      - MASTERMIND_API_HOST=localhost
+      - MASTERMIND_API_PORT=3000
+      - MASTERMIND_OAUTH_URI=account.lab.fiware.org
+      - MASTERMIND_OAUTH_CLIENT_ID=cb19b84cbc654af2bdedaab30dc49f38
```

Comment:
Maybe we can leave here a comment like: "Replace me" or something?

Reply:
@taliaga Added that and a few others